easybuild/easyconfigs/l/libpsml/libpsml-1.1.7-foss-2016b.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'libpsml'
+version = '1.1.7'
+
+homepage = 'https://launchpad.net/libpsml'
+description = """LibPSML provides a Fortran API to parse files in the
+ PSeudopotential Markup Language (PSML) format."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['https://launchpad.net/libpsml/trunk/1.1/+download/']
+
+checksums = ['3f3189d71ec1ce83e00e477ebbf6c516']
+
+dependencies = [
+    ('xmlf90', '1.5.3'),
+]
+
+sanity_check_paths = {
+    'files': ['include/m_psml.mod', 'lib/libpsml.a'],
+    'dirs': ['include', 'lib'],
```

Comment:
Checking for non-empty dirs is a bit silly if you're already checking for specific files in those directories

Reply:
Sure, that was a leftover from previous versions of the library. Fixed.